quickwit-core/src/indexing/split.rs

```diff
@@ -57,7 +57,7 @@ pub struct Split {
     /// The split metadata.
     pub metadata: SplitMetadata,
     /// The local directory hosting this split artifacts.
-    pub local_directory: PathBuf,
+    pub split_scratch_dir: Arc<TempDir>,
```

Comment:
It can be removed after the directory has been published.

Code-wise, the directory is deleted when the TempDir instance is dropped. (if it fails the error is invisible to the user, which is not great)

Reply:
ok got it, that's totally ok